api/src/services/eml-service.ts

```diff
@@ -653,6 +625,26 @@ export class EmlService extends DBService {
     return additionalMetadata;
   }
 
+  async _buildPartnershipMetadata(projectData: IGetProject): Promise<any> {
+    const stakeholders = projectData.partnerships.stakeholder_partnerships;
+    const codes = await this.codes();
+    const indigenous = projectData.partnerships.indigenous_partnerships;
+    const names = codes.first_nations.filter((code) => indigenous.includes(code.id)).map((code) => code.name);
+
+    const sorted = [...names, ...stakeholders].sort((a, b) => a.localeCompare(b));
```

Comment:
Couple of super minor nits:

`names` might be better as `firstNationsNames` or similar, for clarity.
`sorted` might be better as `sortedPartnershipNames` or similar, for clarity.

I'm not super versed on `localeCompare`, but does it do anything different from plain `.sort()` if you don't provide a locale parameter to add language specific sorting rules?

Reply:
so the reason I didn't just leave the default sort was because of a code smell saying "sort will be alphabetical by default" but that's what we're after so I'm going to remove the local compare to avoid confusion and just ignore the smell on this line.